src/lib/metrics_extractor/base_metrics_extractor.py

```diff
@@ -54,26 +51,8 @@ def get_aws_service_client(self, aws_client_name: str = None):
             aws_client_name if aws_client_name else self.aws_client_name
         )
 
-    def get_last_update_time(self, timestream_query_client) -> datetime:
-        """
-        Get time of this entity's data latest update (we append data since that time only)
-        """
-        queryRunner = TimeStreamQueryRunner(
-            timestream_query_client=timestream_query_client
-        )
-
-        # check if table is empty
-        if queryRunner.is_table_empty(
-            self.timestream_db_name, self.timestream_metrics_table_name
-        ):
-            return None
-
-        query = f'SELECT max(time) FROM "{self.timestream_db_name}"."{self.timestream_metrics_table_name}" WHERE {self.RESOURCE_NAME_COLUMN_NAME} = \'{self.resource_name}\''
-        last_date = queryRunner.execute_scalar_query_date_field(query=query)
-        return last_date
-
     @abstractmethod
-    def prepare_metrics_data(self, since_time: datetime) -> (list, dict):
+    def prepare_metrics_data(self, since_time: datetime) -> (list, dict):  # type: ignore
```

Comment:
just for my understanding, could you please elaborate on the benefits of using # type: ignore here instead of annotating as tuple[list, dict]?

Reply:
no benefits :) fixed. good catch, thanks.